Source/UI/IdentityUI.swift

```diff
@@ -23,6 +24,9 @@ public enum LoginMethod {
     /// asks for identifier and then a password to either login or signup if not already registered; the specified email will appear pre-filled in the identity
     /// UI, yet the user will still be able to modify it before submitting.
     case passwordWithPrefilledEmail(EmailAddress)
+    /// prompts the user for using their shared web credentials (if available)
+    @available(iOS 13.0, *)
+    case sharedWebCredentials
```

Comment:
Shouldn't these be called just `sharedCredentials` -- why are they `web`?

Reply:
No, same naming convention as #314